includes/cf-descriptions/dolby-vision.md

```diff
@@ -0,0 +1,4 @@
+**Dolby Vision**<br>
+
+Dolby Vision is a dynamic HDR format.
+It potentially offers a better visual experience compared to HDR10 for several reasons. Firstly, it employs dynamic metadata, allowing for varying brightness levels within HDR content and enabling filmmakers to have more precise control over the image appearance. HDR10 relies on static metadata, limiting the HDR effect to be determined only on a per-movie or per-show basis.
```

Comment:
```suggestion
Dolby Vision is a dynamic HDR format.
It can offer better visual experiences relative to HDR10. The most significant difference is that it uses dynamic metadata which allows varying brightness levels within HDR content which enables filmmakers to have more precise control over the overall image appearance. HDR10 relies on static metadata which limits the HDR effect to a per-movie or per-episode basis.```